### ISSUE TITLE
Add /team page with contact information & foaf profiles

### ISSUE THREAD

**Issue by acka47** [hbz/lobid#325]
We decided to link to the /team page in the "information" (i) tab and display general contact information (a reworked version of http://lobid.org/contact) there as well as a list of team members linking to each team member's profile page.

The profile pages will be reworked versions of the existing ones, e.g. http://lobid.org/team/fs. We will use `http://lobid.org/team/{initials}#!` as the person ID and `http://lobid.org/team/{initials}` as the page URL (meaning what once was the page URL will be the person URL but hey...). Profiles will be put as `.jsonld` files into https://github.com/hbz/lobid/tree/master/conf.

**Comment by acka47** [comment]
Of course, we should redirect http://lobid.org/contact to http://lobid.org/team when this is done.

**Comment by fsteeg** [comment]
Rebased `325-team` branch with my changes on `325-AddTeamInfo`, deleted `325-AddTeamInfo`.

**Comment by fsteeg** [comment]
Current profile pages on staging:

http://stage.lobid.org/team/ap
http://stage.lobid.org/team/pc
http://stage.lobid.org/team/fs

@dr0i: I used your avatar picture as `image` because the other one was so small, you can push a change to the `325-team` branch if you want to change it.

@ChristophEwertowski: You'll need a file called `ce.json` with content like https://github.com/hbz/lobid/blob/325-team/conf/team/ap.json, you can push it to the `325-team` branch.

Next up for myself: set up the actual team page with links to these profiles.

**Comment by acka47** [comment]
Looks good. I think we should talk on Thursday next week about which additional information to add and show in the HTML

**Comment by dr0i** [comment]
+1

**Comment by fsteeg** [comment]
Deployed Christoph's profile to: http://stage.lobid.org/team/ce

@acka47 @ChristophEwertowski: please include a link or `#` with issue number (e.g. `#325`) in your commit messages to make them show up here in the issue. This makes it much easier to see what's happening, as it collects all related commits in one place.

@acka47: beside the question of what to show in the HTML, we might also want to discuss what to add to the data, e.g. for this context here `jobTitle` might make sense. Also something to replace the old `publications` field (maybe more general?). But perhaps all of that is a topic for a separate issue, after we have set up the general team and profile pages, instead of delaying that for at least one week.

**Comment by acka47** [comment]
> beside the question of what to show in the HTML, we might also want to discuss what to add to the data, e.g. for this context here `jobTitle` might make sense. Also something to replace the old `publications` field (maybe more general?).

Yes, that is what I meant with "which additional information to add".

> But perhaps all of that is a topic for a separate issue, after we have set up the general team and profile pages, instead of delaying that for at least one week.

+1

One last thing: I don't know whether we need the "works for" content shown in the HTML. If yes, we should add it as an object with label and show the label with an underlying link.

**Comment by fsteeg** [comment]
Deployed team page to: http://stage.lobid.org/team, with redirect from: http://stage.lobid.org/contact.

The page is not localized for German yet, I will add translation after +1 for the English content.

**Comment by acka47** [comment]
I am  not sure about the layout, it is all looking very dense. It is probably best to talk this through on Friday, especially as I don't have any more time today.. 

Some raw ideas:

- list team members with bulleted list?
- just use the icons with tool tip for twitter, github etc.
- Also, I'd rephrase the first sentence.

**Comment by fsteeg** [comment]
Re latest changes @acka47: I think roleName should be something like "Software developer", not "Software development", see https://schema.org/roleName

And since what I'm talking about is completely invisible when looking at this issue, let me quote myself:

> @acka47: please include a link or # with issue number (e.g. #325) in your commit messages to make them show up here in the issue. This makes it much easier to see what's happening, as it collects all related commits in one place.

**Comment by fsteeg** [comment]
Deployed new version, using the JSON data for the team:

http://stage.lobid.org/team

Note that since the member links are taken from the file, they point to the production version.

**Comment by fsteeg** [comment]
Redeployed after offline dscussion with @acka47: http://stage.lobid.org/team

**Comment by acka47** [comment]
I like this much better but we have to unify the style of the icons:

- a white icon should be used for email (e.g. something like https://thenounproject.com/term/email/12462/). - this mailman logo will probably look a bit better: https://i-mscp.net/index.php/Attachment/922-mailman-jpg/ 
- There will also be a better blog/rss icon somewhere, e.g. https://commons.wikimedia.org/wiki/File:Generic_Feed-icon.svg will probably look better

**Comment by fsteeg** [comment]
Redeployed with updated icons: http://stage.lobid.org/team

**Comment by acka47** [comment]
Looks much better now. +1
